File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Adma/ADMS2015U00GroupHospitalInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Adma
{
    [Serializable]
	public class ADMS2015U00GroupHospitalInfo
	{
		private String _admsGroupId;
		private String _grpSeq;
		private String _hospCode;
		private String _selectFlg;
		private String _grpId;
		private String _grpNm;
		private String _dataRowState;

		public String AdmsGroupId
		{
			get { return this._admsGroupId; }
			set { this._admsGroupId = value; }
		}

		public String GrpSeq
		{
			get { return this._grpSeq; }
			set { this._grpSeq = value; }
		}

		public String HospCode
		{
			get { return this._hospCode; }
			set { this._hospCode = value; }
		}

		public String SelectFlg
		{
			get { return this._selectFlg; }
			set { this._selectFlg = value; }
		}

		public String GrpId
		{
			get { return this._grpId; }
			set { this._grpId = value; }
		}

		public String GrpNm
		{
			get { return this._grpNm; }
			set { this._grpNm = value; }
		}

		public String DataRowState
		{
			get { return this._dataRowState; }
			set { this._dataRowState = value; }
		}

		public ADMS2015U00GroupHospitalInfo() { }

		public ADMS2015U00GroupHospitalInfo(String admsGroupId, String grpSeq, String hospCode, String selectFlg, String grpId, String grpNm, String dataRowState)
		{
			this._admsGroupId = admsGroupId;
			this._grpSeq = grpSeq;
			this._hospCode = hospCode;
			this._selectFlg = selectFlg;
			this._grpId = grpId;
			this._grpNm = grpNm;
			this._dataRowState = dataRowState;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/adma/ADMS2015U00GroupHospitalInfo.java
package nta.med.data.model.ihis.adma;

import java.math.BigDecimal;

public class ADMS2015U00GroupHospitalInfo {
	private Integer admsGroupId;
	private Integer grpSeq;
	private String hospCode;
	private BigDecimal selectFlg;
	private String grpId;
	private String grpNm;
	private String dataRowState;
	public ADMS2015U00GroupHospitalInfo(Integer admsGroupId, Integer grpSeq,
			String hospCode, BigDecimal selectFlg, String grpId, String grpNm,
			String dataRowState) {
		super();
		this.admsGroupId = admsGroupId;
		this.grpSeq = grpSeq;
		this.hospCode = hospCode;
		this.selectFlg = selectFlg;
		this.grpId = grpId;
		this.grpNm = grpNm;
		this.dataRowState = dataRowState;
	}
	public Integer getAdmsGroupId() {
		return admsGroupId;
	}
	public void setAdmsGroupId(Integer admsGroupId) {
		this.admsGroupId = admsGroupId;
	}
	public Integer getGrpSeq() {
		return grpSeq;
	}
	public void setGrpSeq(Integer grpSeq) {
		this.grpSeq = grpSeq;
	}
	public String getHospCode() {
		return hospCode;
	}
	public void setHospCode(String hospCode) {
		this.hospCode = hospCode;
	}
	public BigDecimal getSelectFlg() {
		return selectFlg;
	}
	public void setSelectFlg(BigDecimal selectFlg) {
		this.selectFlg = selectFlg;
	}
	public String getGrpId() {
		return grpId;
	}
	public void setGrpId(String grpId) {
		this.grpId = grpId;
	}
	public String getGrpNm() {
		return grpNm;
	}
	public void setGrpNm(String grpNm) {
		this.grpNm = grpNm;
	}
	public String getDataRowState() {
		return dataRowState;
	}
	public void setDataRowState(String dataRowState) {
		this.dataRowState = dataRowState;
	}

}
